### PR TITLE
TD-1392 Fixed mobile view issue on cookie policy page

### PR DESCRIPTION
**Pull request by manishagarwalnhs**
### JIRA link
https://hee-tis.atlassian.net/browse/TD-1392

### Description
Points addressed in this Commit :
1) Fixed mobile view issue on cookie policy page.

### Screenshots
Digital-Learning-Solutions-Cookie-policy-Digital-Learning-Solutions
WavePlugin

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors (see [info on Text Editor settings](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546185813/DLS+Dev+Process) to avoid whitespace changes)
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript
- [x] Tested any Views or partials created or changed with [Wave Chrome plugin](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related) and addressed any valid accessibility issues
- [ ] Updated/added documentation in [Confluence](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546939432/DLS+Code) and/or [GitHub Readme](https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/README.md). List of documentation links added/changed:
  - [doc_1_here](link_1_here)
- [x] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken
- [x] Scanned over my pull request in GitHub and addressed any warnings from the GitHub Build and Test checks.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml [hunk 1]
```diff
@@ -18,7 +29,7 @@
                     <p>We'll only use these cookies if you say it's OK. We'll use a cookie to save your settings.</p>
 
                     <h3>Tell us if we can use analytics cookies</h3>
-                    <fieldset class="nhsuk-fieldset" id="cookie-statistics-form" aria-describedby="cookie-statistics">
+                    <fieldset class="nhsuk-fieldset" id="cookie-statistics" aria-describedby="cookie-statistics">
```
Comment (rshrirohit) [inline DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml:32]: To tackle the error thrown by wav plugin, you have used a different ID as the existing one was not valid. What's the source/where did you get this new ID from? Are we sure about this accessibility change?

**Review by rshrirohit — COMMENTED** [review]
Please see inline comment on code regarding wave error tackling

File: DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml (hunk 1 above)
Comment (manishagarwalnhs) [reply DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml:32]: The wave plugin checks for the validity of attributes and in this case the aria-describedby has no referenced element in the html. Hence it was throwing an error. The id is neither used in the js nor it is used to prepare client side script on server side. I did not get this id from anywhere but just making the already existing html correct. I'm not sure about the source you are asking. If you still have doubts then please let me know what do you mean by source and I'll look into it.

Comment (rshrirohit) [reply DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml:32]: The id used currently should not be used here. It currently means that it is describing itself (aria-describedby is same as id). 
In order to solve the wave error, we should either try and remove the fieldset altogether OR try to add the id in the h3 heading (tell us.......cookies) and use that id in the fieldset.

**Review by rshrirohit — CHANGES_REQUESTED** [review]

File: DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml (hunk 1 above)
Comment (manishagarwalnhs) [reply DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml]: This is addressed.

**Pull request merged**